assets/src/blocks/Accordion/AccordionEditorScript.js

```diff
@@ -0,0 +1,59 @@
+import { AccordionEditor } from './AccordionEditor';
```

Comment:
Not sure why the other ones are seen as a rename, while this one as a removal and addition :thinking:

Reply:
Maybe I forgot to use `git mv` for this one... 🤔

Reply:
`git mv` shouldn't make a difference, since Git detects whether it's a name "after the fact". `git mv` is just a shortcut for deleting the old file and staging the new one.

Reply:
Maybe spacing in the file changed? When deleting a file, Git will look for exact matches of lines and past a certain threshold it will be regarded as a rename.

Reply:
Oh wait, it's because the `registerAccordionBlock` function (that was wrapping almost everything) is gone, as it's not needed in this approach, it runs `registerBlockType` directly.

Reply:
It's because you also made a change to that file. Ideally you should create an initial commit with just all the renames/moves and then add the changes to a separate commit.

Reply:
Right, I did that in the Carousel PR.